velox/core/QueryCtx.h

```diff
@@ -132,6 +134,14 @@ class QueryCtx {
     }
   }
 
+  folly::Executor* executor0() const {
```

Comment:
why this method called `executor0`

Reply:
`0` added to mark that this method is private.

If that's confusing I would make it "executorOrNull()".